src/main/resources/application.properties

```diff
@@ -365,4 +365,20 @@ task.mirror.epss.lockAtLeastForInMillis=3000
 task.componentIdentification.lockAtMostForInMillis=900000
 task.componentIdentification.lockAtLeastForInMillis=3000
 task.ldapSync.lockAtMostForInMillis=900000
-task.ldapSync.lockAtLeastForInMillis=3000
\ No newline at end of file
+task.ldapSync.lockAtLeastForInMillis=3000
+
+task.cron.metrics.portfolio=* 0/4 * * *
+task.cron.metrics.vulnerability=* 0/4 * * *
+task.cron.mirror.github=* 0/4 * * *
+task.cron.mirror.osv=* 0/4 * * *
+task.cron.mirror.nist=* 0/4 * * *
+task.cron.componentIdentification=* 0/6 * * *
+task.cron.ldapSync=* 0/6 * * *
+task.cron.vulndbSync=* 0/4 * * *
+task.cron.repoMetaAnalysis=* 0/8 * * *
+task.cron.vulnAnalysis=* 0/8 * * *
+task.cron.vulnScanCleanUp=* 0/8 * * *
+task.cron.fortify.ssc.sync=* 0/8 * * *
+task.cron.defectdojo.sync=* 0/8 * * *
+task.cron.kenna.sync=* 0/8 * * *
+task.cron.index.consistency.check=* 0/8 * * *
```

Comment:
For the default values we should add comments to explain what they are, as cron expressions can be quite cryptic if you're not used to them. For example, for `task.cron.metrics.portfolio`, add a comment saying `Every 4 hours`.

Reply:
All good comments.. was going to address all of them in next commit. Thanks for the review!